### DISCUSSION TITLE
posts/nju-program-analysis-a2/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#14] [Announcements]
# posts/nju-program-analysis-a2/

南大软分实验笔记｜A2 常量传播与Worklist求解器

https://blog.wohin.me/posts/nju-program-analysis-a2/

**Comment by zhidu-paxl** [comment]
感谢分享。关于隐藏测试用例的问题，应该主要是需要考虑第一个操作数为NAC的%/0异常，然后应该就可以通过啦。可以参考这个https://github.com/MirageLyu/Tai-e-assignments/commit/a1b827267773a3f0b40875a01a37f3d75608c2f2#diff-0de033441703a7c75d3092bebfa08ac64ee9e7a8b88e66d7e62872f7ece75677

**Reply by brant-ruan** [reply]
谢谢你呀，很有帮助！
